tests/tests_twobody/test_propagation.py

```diff
@@ -406,3 +414,28 @@ def test_propagation_custom_body_works():
     # See https://github.com/poliastro/poliastro/issues/649
     orbit = Orbit.circular(Moon, 100 * u.km)
     orbit.propagate(1 * u.h)
+
+
+@pytest.mark.parametrize(
+    "propagator_coe",
+    [danby_coe, markley_coe, pimienta_coe, mikkola_coe, farnocchia_coe, gooding_coe],
+)
+def test_propagate_with_coe(propagator_coe):
+
+    period = iss.period
+    a, ecc, inc, raan, argp, nu = iss.classical()
+    p = a * (1 - ecc ** 2)
+
+    # Delete de units
+    a = a.to_value(u.km)
```

Comment:
It's `p` the value we need to strip the units to call the función later! That's why the coverage check is failing